src/Domain/User/UserWithPermissions.ts

```diff
@@ -0,0 +1,9 @@
+export type UserWithPermissions = {
+  uuid: string,
+  permissions: UserPermissions,
+}
+
+export type UserPermissions = UserPermission[]
+
+// todo: agree on what should be returned from auth
+export type UserPermission = 'read' | 'write'
```

Comment:
one type/class/interface per file

Reply:
Let's discuss here: https://github.com/standardnotes/files/pull/3#discussion_r598782928

This can be extracted if `UserPermission` gets more complicated. 

We should determine this discussing this `todo` comment and another related one: https://github.com/standardnotes/files/pull/3#discussion_r598729249

Created a task for that here: https://app.asana.com/0/1199914526147392/1200093443903077/f